x/supply/keeper/querier.go

```diff
@@ -0,0 +1,58 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/desmos-labs/desmos/v3/x/supply/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+func NewQuerier(k Keeper, legacyQuerierCdc *codec.LegacyAmino) sdk.Querier {
```

Comment:
Legacy querier could be removed safely. See #497

Reply:
wee need it for the REST endopoints @RiccardoM